Dockerfile.demo

```diff
@@ -1,24 +1,10 @@
-FROM node:18-alpine3.18 as web
-WORKDIR /usr/src/paisa
-COPY package.json package-lock.json* ./
-RUN npm install
-COPY . .
-RUN npm run build
+FROM docker.io/ananthakumaran/paisa:latest
```

Comment:
leave the demo Dockerfile as it is, I build it locally and push it to fly.io without doing a paisa release.